pkgs/development/python-modules/euclid3/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, lib
+, fetchPypi
+}:
+buildPythonPackage rec {
+  pname = "euclid3";
+  version = "0.01";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-JbgnpXrb/Zo/qGJeQ6vD6Qf2HeYiND5+U4SC75tG/Qs=";
+  };
+
```

Comment:
Are there tests to run?

Reply:
No, there aren't any tests.

Reply:
Marking this resolved. Please reopen if you have a better idea for what to do here!